docs-v2/customize/guides/advanced/validate-input-and-output.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: 'Validate input and output'
+sidebarTitle: 'Validate input and output'
+description: 'How to automatically validate your input and ouput with JSON schema and custom alternatives'
+---
+
+Nango provides a straightforward way to validate your data using automatically generated schema files. This guide will walk you through the essentials of data validation in Nango.
```

Comment:
Same here, I wouldn't take about the internals in the intro. 

I would say something along the lines of: Nango automatically validates your integration inputs & outputs. It also offers ways to further customize data validation in code. The guide will walk you through each approach.